Library/Homebrew/descriptions.rb

```diff
@@ -1,7 +1,11 @@
 require "formula"
 require "formula_versions"
+require "search"
+require "searchable"
 
 class Descriptions
+  extend Homebrew::Search
+
   CACHE_FILE = HOMEBREW_CACHE + "desc_cache.json"
 
   def self.cache
```

Comment:
Feel free to punt on this in this PR but I'd love to see this use the `CacheStore` class at some point rather than the weird JSONn cache it current uses.

Reply:
Yeah, I think this PR is big enough already.